src/mlpack/bindings/cli/third_party/CLI/CLI11.hpp

```diff
@@ -447,10 +447,10 @@ CLI11_INLINE std::vector<std::string> compute_win32_argv() {
 namespace enums {
 
 /// output streaming for enumerations
-template <typename T, typename = typename std::enable_if<std::is_enum<T>::value>::type>
+template <typename T, typename = std::enable_if_t<std::is_enum_v<T>>>
```

Comment:
Could you revert the changes done to this file ? while I agree what is being done, but this is not mlpack and not related to the project, this is only a third-party library that we are using.
If you want, please open a PR in their repository directly, if they agree on these changes then the changes will be made their, mlpack pulls the latest version automatically when there is a new release.

Reply:
Indeed, wasn't on purpose and I didn't pay too much attention for third party library code. Reverted the changes in `catch.hpp` as well.